eq-author/cypress/integration/authenticated/builder_spec.js

```diff
@@ -264,42 +264,6 @@ describe("builder", () => {
     cy.get(testId("btn-add-intro"));
   });
 
-  it("can undelete a section introduction", () => {
```

Comment:
I don't think this is the only undelete cypress test

Reply:
Ive done a search for undelete and cant seem to find any others

Reply:
You're right Ive found some that don't use the word 'undelete'